src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySourceUtils.java

```diff
@@ -172,17 +176,17 @@ public static void deleteBigQueryTemporaryTable(Configuration configuration, Big
    *
    * @param configuration Hadoop Configuration.
    * @param bucket the bucket name
-   * @param runId the run ID
+   * @param bucketPath the bucket path to use.
```

Comment:
I don't think it's a path. It's used as as part of  path in one place, as name in other. In both cases I don't think it has part elements (separators)

Reply:
It does not at this time. but I want to keep this generic for the future (anything could be used here as long as it is a valid GCS name)

Reply:
That's what I mean. Don't name it path as someone can pass a path and it won't work as bucket name

Reply:
I've reverted this change to match RunID. It is true that we only use this for now.